packages/@aws-cdk/aws-kms/lib/key.ts

```diff
@@ -389,41 +403,33 @@ export class Key extends KeyBase {
     }
   }
 
-  private allowAccountIdentitiesToControl() {
+  /**
+   * Adds the default key policy to the key. This policy gives the AWS account (root user) full access to the CMK,
+   * which reduces the risk of the CMK becoming unmanageable and enables IAM policies to allow access to the CMK.
+   * This is the same policy that is default when creating a Key via the KMS API or Console.
+   * @see https://docs.aws.amazon.com/kms/latest/developerguide/key-policies.html#key-policy-default
+   */
+  private grantDefaultKeyPolicy() {
```

Comment:
I don't love this name, seems a little too generic. Also, the `grant` word is weird in this context, as it's more of a setter than a `grant` (in the CDK-specific meaning of the word `grant`).

Maybe `addKmsStarForRootPrincipalToResourcePolicy()`? I'm open to other suggestions too.

Reply:
+1 on not using "grant" here to avoid confusion.  What's wrong with `allowAccountIdentitiesToControl`?